### DISCUSSION TITLE
Can't download the latest version(s) of Leapp

### DISCUSSION THREAD

**Discussion by gomezga1** [Noovolari/leapp#538] [General]
Hi,

It seems like there is an issue with the permissions of the S3 bucket in which you placed the latest version(s) of Leapp, since I'm not able to download them as I get an access denied error.

Best,

Alejandro.

**Comment by ericvilla** [comment]
You're right @gomezga1, thank you very much for reporting!
I'm going to fix it now.

**Comment by ericvilla** [comment]
Fixed!

**Reply by gomezga1** [reply]
Thanks! :)
